app/controllers/MoviesGallery.scala

```diff
@@ -0,0 +1,24 @@
+package controllers
+
+import akka.http.scaladsl.model.HttpHeader.ParsingResult.Ok
+import controllers.movieDatabase
+import javax.inject.Inject
+import play.api._
+import play.api.mvc.{AbstractController, ControllerComponents}
+import play.modules.reactivemongo.{MongoController, ReactiveMongoApi, ReactiveMongoComponents}
+import play.mvc.Controller
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243893)

Reply:
rip